packages/devtools_app/lib/src/flutter/split.dart

```diff
@@ -64,15 +63,21 @@ class Split extends StatefulWidget {
   /// The minimum size each child is allowed to be.
   final List<double> minSizes;
 
+  /// Splitter widgets to divide [children].
+  ///
+  /// If this is null, a default splitter will be used to divide [children].
+  final List<SizedBox> splitters;
+
   /// The key passed to the divider between children[index] and
   /// children[index + 1].
   ///
   /// Visible to grab it in tests.
   @visibleForTesting
   Key dividerKey(int index) => Key('$this dividerKey $index');
 
-  /// The size of the divider between children in logical pixels (dp, not px).
-  static const double dividerMainAxisSize = 10.0;
+  /// The default size of the divider between children in logical pixels (dp,
```

Comment:
no need to indicate dp, not px as all calculations we do for flutter should be in logical pixels.

Reply:
done.